packages/@okta/vuepress-site/docs/guides/common-hook-set-up-steps/main/index.md

```diff
@@ -65,6 +75,165 @@ After including the `npm` packages, add the following code snippet in your proje
 
 <StackSelector snippet="auth" noSelector/>
 
+### OAuth 2.0: Client Secret
+
+The OAuth 2 Client Secret method sends a signed JWT to your external service. To use this method, you must make the following configurations to your org, and add code to decode the JWT from the Okta inline hook call:
+
+* Create an app integration
+* Add a custom scope
+* Add OAuth 2.0 authentication fields to your inline hook
+* Add code to verify the JWT
+
+#### Create an app integration
+
+Before you can implement authorization, you need to register your app in Okta by creating an app integration from the Admin Console.
+
+1. In the Admin Console, navigate to **Applications** > **Applications**.
+1. Click **Create App Integration**.
+1. Select **API Services** as the Sign-in method.
+1. Click **Next**.
+1. Specify the app integration name, then click **Save**.
+1. From the General tab of your app integration, note your generated **Client ID** and **Client secret**. These value are used when configuring your inline hook authentication.
+
+#### Add a custom scope
+
+<StackSnippet snippet="scope"/>
+
+#### Add OAuth 2.0 authentication fields to your inline hook
+
+When creating your inline hook, in the Authentication section, select **OAuth 2.0**.
+1. In the Client Authentication field, select **Use client ssecret** from the dropdown menu.
+1. Add the **Client ID** and **Client Secret** values from your app integration.
+1. Add the authorization server’s token URL, for example `https://${yourOktaDomain}/oauth2/default/v1/token`, and the custom scope you created previously.
```

Comment:
I added "such as" here rather than "for example" because "for example" should be: ", for example, `https....", and that's just a lot of commas haha

```suggestion
1. Add the authorization server’s token URL, such as `https://${yourOktaDomain}/oauth2/default/v1/token`, and the custom scope that you created previously.
```